apps/openmw/mwgui/spellwindow.cpp

```diff
@@ -45,7 +45,7 @@ namespace MWGui
         getWidget(mEffectBox, "EffectsBox");
         getWidget(mFilterEdit, "FilterEdit");
 
-        mFilterEdit->setUserString("IgnoreTabKey", "y");
+        mFilterEdit->setUserString("AcceptTab", "true");
```

Comment:
Probably it would be better to set such strings in layout files rather than hardcoding them.

Reply:
Done.